layouts/partials/sidebar/docs-toc-mobile.html

```diff
@@ -1,6 +1,6 @@
 {{ if and (ne .Params.toc false) (partial "private/has-headings.html" .) -}}
   <details>
-    <summary>On this page</summary>
+    <summary>{{ i18n "on-this-page" }}</summary>
```

Comment:
We recently harmonized all i18n keys to `snake_case`, see https://github.com/gethyas/doks-core/blob/main/i18n/en.toml

So it should be `i18n "on_this_page"`.

Reply:
Adjusted 👍 . Sry was using older version